src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/ReportLatestTests.kt

```diff
@@ -0,0 +1,51 @@
+package org.vaccineimpact.orderlyweb.tests.integration_tests.tests.web
+
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.vaccineimpact.orderlyweb.models.Scope
+import org.vaccineimpact.orderlyweb.models.permissions.ReifiedPermission
+import org.vaccineimpact.orderlyweb.tests.integration_tests.tests.IntegrationTest
+import org.vaccineimpact.orderlyweb.test_helpers.insertReportWithPublishedAndUnpublishedVersions
+import org.vaccineimpact.orderlyweb.tests.integration_tests.WebPermissionChecker
+
+class ReportLatestTests : IntegrationTest()
+{
+    @Test
+    fun `reviewer can get latest unpublished version`()
+    {
+        val reportName = "test"
+        val publishedVersionId = "20211001-013233-72d597e9"
+        val unpublishedVersionId = "20211002-013233-72d597e9"
+        insertReportWithPublishedAndUnpublishedVersions(reportName, publishedVersionId, unpublishedVersionId)
+
+        val url = "/report/$reportName/latest"
+        val permissions = setOf(
+            ReifiedPermission("reports.review", Scope.Global()), 
+            ReifiedPermission("reports.read", Scope.Global())
+        )
+        val checker = WebPermissionChecker(url, permissions)
+        val response = checker.requestWithPermissions(permissions)
+        Assertions.assertThat(response.statusCode).isEqualTo(200)
+        Assertions.assertThat(response.text).contains("$unpublishedVersionId\" selected")
```

Comment:
There might be a more elegant way of testing this. Let me know if you have any ideas

Reply:
Some of the other tests use Jsoup to parse the HTML. e.g.  https://github.com/vimc/orderly-web/blob/master/src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/VersionPageTests.kt#L34

You could try that, and e.g. use css selector to pick the text of the crumb-item link, or the report id just under the header, or the selected option as you are doing here. Might be nice to include a comment on which part of the page you are picking if it isn't obvious from the selector.